Gopkg.toml

```diff
@@ -6,12 +6,10 @@ ignored = ["github.com/ethereum/go-ethereum/whisper/notifications"]
 
   [[prune.project]]
     name = "github.com/karalabe/hid"
-    non-go = false
```

Comment:
maybe it would be better to add non-go=true as a default for other packages and keep false for hid and geth?

Reply:
That seems good. I will test it.

Reply:
Yeah, I thought it is true by default. But let's be explicit.